src/test/java/edu/kis/powp/jobs2d/gui/extensions/ModifierListener.java

```diff
@@ -6,9 +6,8 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
-public class ModifierListener implements ActionListener
+public class ModifierListener extends ActionTemplate implements ActionListener
 {
-    private boolean enabled;
     private TransformationModifier modifier;
     private TransformingDriver lineTrans, specialLineTrans;
```

Comment:
I have expected a list of TransformingDrivers. Maybe the list could be static, so it can be accessed by all ModifierListener instances?